blockbase/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "blockbase",
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
Would like to pull out the version changes from this change.  There shouldn't be a reason for those right?  Just something committed testing out the script?